### PR TITLE
Add Workshop model

### LINKED ISSUE #9: Workshop model
In order to show users available workshops
For customers
We need to add workshop model

### PR DESCRIPTION
**Pull request by msvrtan**
In order to show users available workshops
For customers
We need to add workshop model

Closes #9

### CODE REVIEW

File: src/Entity/Workshop.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    {
+        return $this->description;
+    }
+
+    public function setDescription(string $description)
+    {
+        $this->description = $description;
+    }
+
+    public function getCreatedAt(): DateTime
```
Comment (flackovic) [inline src/Entity/Workshop.php:92]: We need setCreatedAt too @msvrtan

**Review by zgperak — APPROVED** [review]
This looks OK

**Review by flackovic — CHANGES_REQUESTED** [review]
Add setCreatedAt

File: src/Repository/WorkshopRepository.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class WorkshopRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Workshop::class);
+    }
+
+    /*
```
Comment (bephrimast) [inline src/Repository/WorkshopRepository.php:18]: Keep commented-out code on the repository?

**Review by HcsOmot — APPROVED** [review]
👍

File: src/Entity/Workshop.php (hunk 1 above)
Comment (msvrtan) [reply src/Entity/Workshop.php:92]: Actually we dont want to allow setting it from the outside of the entity. It's value is set in __constructor() and should not be changed

**Review by franraknic — APPROVED** [review]

**Comment by div3r** [comment]
sve 5

**Review by flackovic — APPROVED** [review]
:+1:

**Review by div3r — APPROVED** [review]
👍

**Review by bephrimast — APPROVED** [review]
:+1:

**Pull request merged**